### DISCUSSION TITLE
怎样联合SmartDNS使用呢？

### DISCUSSION THREAD

**Discussion by lon91ong** [SeaHOH/GotoX#162] [问答/咨询]
[SmartDNS](https://github.com/pymumu/smartdns)在我的理解中和 [issues#137](https://github.com/SeaHOH/GotoX/issues/137#issuecomment-540917401)中提到的被删档的Pcap_DNSProxy是一个类型的东东
在这里搜了一下`smartdns`没有结果
不知道能不能配合GotoX使用？应该怎么配合？

**Comment by SeaHOH** [comment]
如果你是用浏览器扩展控制切换网站代理，只需要设置系统代理就可以，否则需要在 [dns/localservers] 设置。

配置基本上都有详细注释，看不明白再提问，我会根据反馈进行补充修改。
